styles.css

```diff
@@ -0,0 +1,597 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  border: 0;
+}
+
+body {
+  background: #fff;
+  box-sizing: border-box;
+  display: grid;
+  grid-template-rows: auto auto auto auto;
+  font-family: 'lato', sans-serif;
+}
+
+.menu {
+  width: 20px;
+  margin-top: 10px;
+}
+
+.button-menu {
+  width: 60px;
+  margin: 20px 0 0 20px;
+  position: absolute;
+  z-index: 100;
+  background: transparent;
+  border-style: none;
+  cursor: pointer;
+}
+
+.menu-close {
+  width: 39%;
+  display: none;
+  margin-top: 12px;
+}
+
+.close-menu-active {
+  display: block;
+}
+
+.nav-mobile {
+  position: fixed;
+  display: none;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  z-index: 99;
+  background-image: linear-gradient(to right, #ec5242, rgb(100, 100, 100), #272a31);
+  color: white;
+  list-style: none;
+  margin-top: 0;
+}
+
+.nav-mobile-active {
+  display: block;
+  height: 100%;
+  padding-top: 60px;
+  width: 100%;
+}
+
+.nav-mobile-list {
+  text-decoration: none;
+  color: #272a31;
+  font-family: 'roboto', sans-serif;
+  font-weight: 600;
+  font-size: 32px;
+}
+
+.nav-mobile-list-parent {
+  padding: 15px;
+}
+
+.container {
+  display: flex;
+  background-image: url('images/grey\ background.jpeg');
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+  height: 35rem;
+}
+
+.introduction {
+  margin: 16px;
+}
+
+.intro-text {
+  padding-top: 120px;
+  color: #ec5242;
+  font-size: 20px;
+}
+
+.intro-text-h1 {
+  font: 900 33px 'roboto', sans-serif;
+  color: #ec5242;
+  margin-bottom: 1.2rem;
+}
```

Comment:
- [ ] As the previous reviewer suggested, kindly ensure that you follow the design guidelines as much as possible. We are very close to achieving a professional heading that matches the design. Kindly ensure the heading covers almost 60% width of the screen size. This will make our website more professional and user-friendly ☺️. 

Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/226098677-099622d3-87d6-4f2e-97a5-6962bcf2b3a0.png)